### DISCUSSION TITLE
Make "Open Link" Dialog/Popup less overwhelming visually

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [ppy/osu#28055] [Ideas]
![image](https://github.com/ppy/osu/assets/25248999/4b695f68-f965-4e95-b6e9-0749424f73e7)
When clicking one of the banners at the bottom in Lazer, this is the popup you get.

While I don't have any direct proposals on what to change, I feel like this dialog is way too overwhelming for just opening a link. Especially considering the average user doesn't fully read and just clicks whatever feels to be the right button often.

I feel like one of the main issues here is there being three buttons, vertically positioned, which is just very heavy visually speaking. I still really like the idea of being able to copy to clipboard.
Potentially it would be an option to move the copying of the url into a little inline clipboard icon-button after the URL? Like the ones you often see on websites are code snippets.

And then maybe - although I don't know if your design identity/system allows that - the remaining two buttons could be positioned horizontally?

*Also, I am very well aware that this is a very small thing, and of course very far down in priority, but especially in the long term, this might still be good to look at, because the many little things make up a lot too*

**Comment by peppy** [comment]
Yeah dialogs are really dated design wise. Not sure if @arflyte has a new version proposed yet, but I definitely agree with horizontal buttons and a less imposing UI.

**Comment by arflyte** [comment]
![Figma_240502-125857](https://github.com/ppy/osu/assets/3632750/ed0aeb2e-4af6-47a6-995f-92ee560d9368)
![Figma_240502-125917](https://github.com/ppy/osu/assets/3632750/df3fa08a-28dc-445d-9005-a8f87466fab7)

There were proposals in the past, but it's far from polished.

**Reply by peppy** [reply]
I guess the lower two look to be most suitable, but I'm not sure of that weird top bar/element alignment.

**Comment by arflyte** [comment]
New Popup Dialog designs:

https://www.figma.com/file/MpxMxwJ53dJo1rE199xYcR/Error-Dialog-2?type=design&node-id=0%3A1&mode=design&t=blhSVkEvpgt6VMNo-1
